Desktop/html/js/utils.js

```diff
@@ -20,60 +21,60 @@ function formatFixed(value, fixIt)
 }*/
 
 
-function formatMoney(_currency='EUR', amount) {
+function formatMoney(_currency='EUR', amount, thousands=true) {
 	const formatter = new Intl.NumberFormat(currentLocaleId, {
 		style: 'currency',
 		currency: _currency,
 		trailingZeroDisplay: 'stripIfInteger',
-		useGrouping: useThousandsSeparators
+		useGrouping: useThousandsSeparators || thousands
 	});
 	
 	return amount == "." ? amount : formatter.format(amount)
 }
 
-function formatFixed(number, digitsFrac, noZeroLead=false) {
+function handleNoZeroLead(number, result, noZeroLead)
+{
+	if(number >= 0 && number < 1 && noZeroLead && !result.startsWith("1")) //we need to check whether it starts with despite number < 1 because of rounding in the formatting
```

Comment:
```suggestion
	if(number >= 0 && number < 1 && noZeroLead && !result.startsWith("1")) //we need to check whether it starts with 1 despite number < 1 because of rounding in the formatting
```